frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -1911,8 +1914,9 @@ void ProgramStructure::createControls() {
         if (ExternConverter::cvtAsGlobal(this, ext.first)) {
             // FIXME -- 'declarations' is a vector Nodes instead of Declarations.  Should
             // FIXME -- fix the IR to use a NameMap for scopes anyways.
-            IR::IndexedVector<IR::Declaration> tmpscope;
-            auto e = ExternConverter::cvtExternInstance(this, ext.first, ext.second, &tmpscope);
+            IR::IndexedVector<IR::Node> tmpscope;
+            auto e = ExternConverter::cvtExternInstance(this, ext.first, ext.second,
+                                                        nullptr, &tmpscope);
```

Comment:
Is passing a nullptr for the local scope argument safe?  That's why the code previously created tmpscope.  Similarly, for the global scope, should just pass `declarations` directly?